lib/Widget/shared/alert_style.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:AbdoCare_Web/page/dashboard_postHos.dart';
+import 'package:AbdoCare_Web/page/dashboard_pre.dart';
+import 'package:flutter/material.dart';
+
+class Dialogs {
+  static Future<void> alertDialogNoData(BuildContext context) async {
+    await showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
```

Comment:
This one is an alert dialog for `กรุณาทำแบบประเมินให้ครบถ้วน`. Try to name it like `alertToCompleteEvalutation` sth like that.

Reply:
thank ka

Reply:
solve in dee4924